javatools_unicode/src/main/java/org/xvm/tool/BuildUnicodeTables.java

```diff
@@ -212,10 +212,16 @@ void writeResult(final String name, final String[] array) throws IOException {
         }
 
         final var sb = new StringBuilder();
-        sb.append(name).append(": [index] \"str\" (freq) \n--------------------");
+        sb.append(name)
+            .append(": [index] \"str\" (freq) \n--------------------");
         int index = 0;
         for (final var entry : map.entrySet()) {
-            sb.append("\n[").append(index).append("] \"").append(entry.getKey()).append("\" (").append(entry.getValue()).append("x)");
+            sb.append("\n[")
+                .append(index)
```

Comment:
ditto

Reply:
And again - this is continuation indentation. They MUST be larger than the indentation level of the beginning of the chain statement, and they MUST be a multiple of the global indentation level.